lib/components/Dialog/Dialog.scss

```
@@ -0,0 +1,4 @@
+.deriv-dialog{
+  width:440px;
```

```suggestion
  min-width:440px;
```
this way the consumer don't need to apply any width if the content inside the Dialog take more width